_pages/en/strategy-build-vs-buy-explained.md

```diff
@@ -17,406 +17,79 @@ permalink: /strategy-build-vs-buy-explained.html
-Part of this problem is with the procurement process.
-With the ability to increase the value of a contract by up to 50% without tendering again, contractors can submit “suicide bids” that they know are much too low.
-This makes the contracted option initially on paper look better when compared with the costs to developing in house.
+### Scope of This Document
 
-Time and again balling balloon on IT contracts have been observed.
-The Phoenix pay project started at $309M and is expected to now cost over $2.6B.
-The Email transformation initiative started at $x and was $100M by the time it was suspended.
-The contract for the content management system solution for the Canada.ca project started at $1.54M and was $14.9M by the end.
+The scope of this document is to provide you with different examples or scenarios which will help you to put the Adopt, Buy, Build strategy into context. It is not meant as a definitive guide. There are many different considerations and many different variables when it comes to decision making around whether to build or buy a solution or a component thereof. As such, we will be making a core set of assumptions that may not be applicable to your precise situation. Please refer to the actual strategy document for more detailed information.
 
-Though IT projects don't just end up over budget due to suicide bids.
-They can end up over budget due to a lack of strong integration with internal policy and subject matter experts on the project team.
-Without the input from these experts, projects end up not delivering on functionality.
-Fixing systems that don't meet functional requirements, comes at a cost, eats into the predicted cost savings for the new software solution.
-ETI was supposed to save $50M/year but was only completed up to 20% so any savings achieved are reduced by at least that amount.
-Coupled with the budget overruns, it would take 10+ years to achieve a return on investment now.
-Phoenix was expected to save $70M a year.
-Even if all those savings have been achieved, the increased cost means that the return on investment will take 37 years.
-The phoenix project is another example of this.
-It was launched with many important pay processing functions missing.
-Had the project been done in house, and/or with a tighter coupling with the subject matter experts, it is likely that a stronger case could have been made to not go forward without those functions.
+### Use Cases
 
-Further, offloading of risk to the taxpayer is not solely about the build decision itself but rather the ongoing operation.
-Often traditional build projects include a significant component of ongoing run and support.
-As such the ongoing support costs, and risk of being required to support companies solely due to severe vendor lock-in.
+Most of the time, we will not only adopt, buy, or build something.
+It's most often a combination of the 3.
```

Comment:
wondering about a way to not have the word 'most' appear in both sentences (this and the one above). Perhaps this could be written as 'Generally the obtaining an IT solution does not involve only adopting or only buying or only building something; it is most often a combination of the 3'.  Or something like that?

Reply:
"We" 

Also 

It's most often a combination of the 3.
==>
Often it is some permutation of the three 

Otherwise we are claiming that adopt + build + buy is the most common, and Im not sure that is the case (versus adopt build, adopt buy, or build buy)